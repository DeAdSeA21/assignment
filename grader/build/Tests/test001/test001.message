Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t9916.58
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
