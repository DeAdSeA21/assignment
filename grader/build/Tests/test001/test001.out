Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	9993.75
2	9987.47
3	9981.17
4	9974.84
5	9968.48
6	9962.09
7	9955.67
8	9949.23
9	9942.76
10	9936.26
11	9929.73
12	9923.17
